go.mod

```diff
@@ -56,7 +56,8 @@ require (
 	k8s.io/cloud-provider v0.16.8
 	k8s.io/code-generator v0.16.8
 	k8s.io/kops v1.15.2
-	k8s.io/kubelet v0.0.0
+	k8s.io/kubelet v0.18.3
```

Comment:
does it matter that we use `0.18` here, `1.15` below and `0.16` elsewhere?

Reply:
`k8s.io/kubelet` doesn't need to be updated. This change only needs `k8s.io/kubernetes v1.15.3`. I was updating some k8s.io packages locally and forgot to revert the kubelet dependency.